v2/ical.NET/Components/Journal.cs

```diff
@@ -14,19 +14,17 @@ public JournalStatus Status
             set { Properties.Set("STATUS", value); }
         }
 
-        private void Initialize()
+        /// <summary>
+        /// Constructs an Journal object, with an iCalObject
+        /// (usually an iCalendar object) as its parent.
+        /// </summary>
+        public Journal()
         {
             Name = Components.Journal;
         }
 
         protected override bool EvaluationIncludesReferenceDate => true;
 
-        protected override void OnDeserializing(StreamingContext context)
```

Comment:
I seem to recall that deleting these overrides can cause subtle problems, so I think these should be put back.

I'm sorry my other comment was unclear: I meant delete the call to `Initialize()` from this method, moving that logic to the constructor, which you've done. Can you put back this override? Then it will be ready to merge.

Reply:
Reverted removal of OnDeserializing()
(see 7a3f471)